juntagrico/entity/share.py

```diff
@@ -10,6 +10,7 @@
 
 class Share(Billable):
     member = models.ForeignKey('Member', blank=True, on_delete=models.PROTECT)
+    value = models.DecimalField(_('Wert'), max_digits=8, decimal_places=2, default=float(Config.share_price()))
```

Comment:
this probably will be evaluated at migrate and not at the actual object creation, see
https://stackoverflow.com/questions/12649659/how-to-set-a-django-model-fields-default-value-to-a-function-call-callable-e
and
https://docs.djangoproject.com/en/3.1/ref/models/fields/#default
i tried it locally and there it reacted to a change in the setting
nontheless i would use a callable just to be sure